### PR TITLE
VBLOCKS-2473 fix logLevel param to accept strings

### PR DESCRIPTION
**Pull request by charliesantos**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

## Pull Request Details

VBLOCKS-2473

### Before review
* [x] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [x] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Squashed erroneous commits if necessary
* [ ] Re-tested if necessary

### CODE REVIEW

File: lib/twilio/device.ts [hunk 1]
```diff
@@ -1882,7 +1885,13 @@ namespace Device {
...
 
     /**
-     * Log level.
+     * Sets the log level.
+     *
+     * Possible values include any of the following numbers:
+     * 0 = trace, 1 = debug, 2 = info, 3 = warn, 4 = error, 5 = silent
+     *
+     * Or any of the following strings:
+     * 'trace', 'debug', 'info', 'warn', 'error', 'silent'
```
Comment (mhuynh5757) [inline lib/twilio/device.ts:1894]: If I recall correctly, the same strings in all-caps are also valid in the `logLevel` library.

Comment (charliesantos) [reply lib/twilio/device.ts:1894]: I checked the types and only numbers and lower case strings are accepted
```ts
declare namespace log {
    /**
     * Log levels
     */
    interface LogLevel {
        TRACE: 0;
        DEBUG: 1;
        INFO: 2;
        WARN: 3;
        ERROR: 4;
        SILENT: 5;
    }

    /**
     * Possible log level numbers.
     */
    type LogLevelNumbers = LogLevel[keyof LogLevel];

    /**
     * Possible log level descriptors, may be string, lower or upper case, or number.
     */
    type LogLevelDesc = LogLevelNumbers
        | 'trace'
        | 'debug'
        | 'info'
        | 'warn'
        | 'error'
        | 'silent'
        | keyof LogLevel;
```

Comment (mhuynh5757) [reply lib/twilio/device.ts:1894]: `keyof LogLevel` would include the all-caps variants of the strings.

Comment (charliesantos) [reply lib/twilio/device.ts:1894]: Ah true! Let me update the doc

**Pull request merged**